Index: src/org/nantes/univ/archi/platform/Loader.java
===================================================================
--- src/org/nantes/univ/archi/platform/Loader.java	(revision 89ae11d5099dc77cce43a0c348158610a5f2ec48)
+++ src/org/nantes/univ/archi/framework/Loader.java	(revision )
@@ -1,17 +1,9 @@
-package org.nantes.univ.archi.platform;
+package org.nantes.univ.archi.framework;
 
-import java.awt.Color;
-import java.awt.Dimension;
-import java.awt.Font;
-import java.awt.Graphics;
 import java.io.FileReader;
 import java.lang.reflect.Method;
 import java.util.Properties;
 
-import javax.swing.JComboBox;
-import javax.swing.JFrame;
-import javax.swing.JPanel;
-
 /**
  * Created by romain on 07/03/17.
  */
@@ -65,6 +57,4 @@
 		// lancer les config en autorun
 
 	}
-
-
 }
Index: src/org/nantes/univ/archi/appli/ArchiPlugin.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/org/nantes/univ/archi/appli/ArchiPlugin.java	(revision 89ae11d5099dc77cce43a0c348158610a5f2ec48)
+++ src/org/nantes/univ/archi/appli/ArchiPlugin.java	(revision )
@@ -1,6 +1,6 @@
 package org.nantes.univ.archi.appli;
 
-import org.nantes.univ.archi.platform.Loader;
+import org.nantes.univ.archi.framework.Loader;
 import org.nantes.univ.archi.plugin.Personne;
 
 /**
